src/app/app.component.ts

```diff
@@ -59,11 +66,20 @@ export class AppComponent implements OnInit, OnDestroy {
   }
 
   onLoginClick() {
-    this.store.dispatch(login());
+    const redirect = '/home';
+    if (!this.loginService.isLogged()) {
+      this.loginService.googleLogin().then(
+          res => this.router.navigate([redirect]));
+    } else {
+      this.router.navigate([redirect]);
+    }
   }
 
   onLogoutClick() {
+    const redirect = '/login';
+    SecurityService.removeAccessToken();
```

Comment:
Even after clicking `Log Out` I don't get prompted for username/password when I click Log In. I also noticed that my login token was saved between runs. Is this the intended behavior? It seems like we should have a way to revoke the token and force the user to re-enter their credentials. If this is working as intended, we can talk about this with the stakeholders; maybe this behavior would be preferred.

Reply:
Yes, the login token is saved when a usr logs in, and is flushed when logged out. At the moment it would be the intended behavior, but we need to perform a log out from google's account besides from our app (which are two different things). Its ok if we create a story to handle this more clearly?

Reply:
Yes, that's OK